action/putpollresult.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2018 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"math/big"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/pkg/keypair"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+	"github.com/iotexproject/iotex-core/pkg/version"
+	"github.com/iotexproject/iotex-core/protogen/iotextypes"
+	"github.com/iotexproject/iotex-core/state"
+)
+
+// PutPollResult represents put the poll result from beacon chain.
+type PutPollResult struct {
+	AbstractAction
+
+	height     uint64
+	candidates state.CandidateList
```

Comment:
Since everyone has the access to the beacon chain states, it seems to not necessary that  we need to pass the list via an action.